blog/2021-12-13-maintainers-interview.md

```diff
@@ -0,0 +1,120 @@
+---
+slug: maintainers-interview
+title: Meet Capact maintainers and learn how to become one!
+authors:
+  - mszostok 
+  - pkosiec
+
+tags:
+  - capact
+  - open-source
+  - go
+  - kubernetes
+  - cloud-native
+  - dry
+  - interview
+  - maintainers
+  - contributors
+  - career
+---
+
+Did you know that it's already been 16 months since the Capact was born?
+
+In the previous blog post, you learned which problems Capact aims to solve. This time, Capact maintainers took some time to answer questions you might have if you're considering contributing to the project.
```

Comment:
In a previous blog post, we discussed some of the problems Capact aims to solve. This time, Capact maintainers took some time to answer questions you might have if you're considering contributing to the project.